editor/editor_resource_picker.cpp

```diff
@@ -858,6 +863,11 @@ void EditorResourcePicker::drop_data_fw(const Point2 &p_point, const Variant &p_
 		}
 
 		edited_resource = dropped_resource;
+
+		if (Input::get_singleton()->is_key_pressed(Key::CMD_OR_CTRL)) {
+			_edit_menu_cbk(OBJ_MENU_MAKE_UNIQUE);
```

Comment:
I just noticed that this will call `_resource_changed()`, so you have to add `else` for the call below.